cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -42,7 +45,22 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
 
   # Run conformance test only at CI and if the provider has them activated
   conformance_config=$DIR/${provision_dir}/conformance.json
+
   if [ "${CI}" == "true" -a -f $conformance_config ]; then
+    sonobouy_plugins=""
+
+    if [ "$RUN_KUBEVIRT_CONFORMANCE" == "true" ]; then
+      LATEST=$(curl -L "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/latest")
+      ${ksh} apply -f "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-operator.yaml"
+      ${ksh} apply -f "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-cr.yaml"
+
+      ${ksh} wait -n kubevirt kv kubevirt --for condition=Available --timeout 15m
+
+      export SONOBUOY_EXTRA_ARGS="--plugin https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/conformance.yaml"
+      hack/conformance.sh $conformance_config
+    fi
+
+    export SONOBUOY_EXTRA_ARGS="--plugin systemd-logs --plugin e2e"
```

Comment:
I think it makes sense to first run the k8s conformance tests before going to kubevirt. What do you think?

Reply:
I thought running kubevirt conformance tests first is better because it take less time 
then e2e so we could fail faster.
What you think?

Reply:
Fine by me.